tests/elm/ElmLintBearTest.py

```diff
@@ -0,0 +1,56 @@
+from bears.elm.ElmLintBear import ElmLintBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+good_file_no_unexpected_comma = """
+module UnexpectedComma (..) where
+var =
+"Hello World!"
+"""
+
+
+bad_file_unexpected_comma = """
+module UnexpectedComma (..) where
+va,r =
```

Comment:
part of the reason to avoid `var` above is that here you seem to be implying that `var` is a thing.

Some languages (notably Python) multiple variable (tuple) assignments are ok, but Elm isnt one of them.
There are much simpler 'unexpected comma' sample files.

But if you want to keep the structure of this sample file, you should use `a, b = \n...`